src/EditorFeatures/Text/ContentTypeNames.cs

```diff
@@ -12,5 +12,8 @@ internal static class ContentTypeNames
         public const string XamlContentType = "XAML";
         public const string JavaScriptContentTypeName = "JavaScript";
         public const string TypeScriptContentTypeName = "TypeScript";
+
+        public const string CSharpLspContentTypeName = "C#_LSP";
```

Comment:
Did something need these in the base layer? Or can we keep this in the LSP layer? It made me feel happy knowing nothing outside that project could use this.

Reply:
as requested here :) https://github.com/dotnet/roslyn/pull/40320#discussion_r357877602

I can attempt to move the ilanguageclient impl to the liveshare layer which would also solve it, but I didn't b/c the LSP client dependencies are already in the VS project.